### ISSUE TITLE
Last dropout is disconnected from fc_audioset layer

### ISSUE THREAD

**Issue by ilic-mezza** [qiuqiangkong/audioset_tagging_cnn#52]
It appears that, in all CNN models, the last dropout, i.e., ```embedding = F.dropout(x, p=0.5, training=self.training)```, is actually disconnected from the output linear layer, i.e., ```self.fc_audioset(x)```. 
Indeed, the ```forward``` method of these models reads:
```
x = F.relu_(self.fc1(x))
embedding = F.dropout(x, p=0.5, training=self.training)
clipwise_output = torch.sigmoid(self.fc_audioset(x))
```

By reading the [arXiv paper](https://arxiv.org/abs/1912.10211), it seems that the last dropout should have instead connected the 2048-embedding layer to the 527-output layer. Indeed, the paper reads:

_"Dropout [38] is applied after each downsampling operation **and fully connected layers** to prevent systems from overfitting."_

Therefore, I expected to see the following:
```
x = F.relu_(self.fc1(x))
embedding = F.dropout(x, p=0.5, training=self.training)
clipwise_output = torch.sigmoid(self.fc_audioset(embedding))
```

Am I missing something?

Thank you,
Alessandro
